publish/dir.props

```diff
@@ -5,6 +5,9 @@
   <PropertyGroup>
     <ContainerName Condition="'$(ContainerName)' == ''">dotnet</ContainerName>
     <NuGetPushTimeoutSeconds Condition="'$(NuGetPushTimeoutSeconds)' == ''">3600</NuGetPushTimeoutSeconds>
+    <BinariesRelativePath>$(Channel)/Binaries/$(SharedFrameworkNugetVersion)</BinariesRelativePath>
+    <InstallersRelativePath>$(Channel)/Installers/$(SharedFrameworkNugetVersion)</InstallersRelativePath>
+    <ChecksumsRelativePath>$(Channel)/Checksums/$(SharedFrameworkNugetVersion)</ChecksumsRelativePath>
```

Comment:
@mellinoe will be working next week on changing our storage structure.  I think we can start making that change with the checksums now.  Sync with Eric M and me about it.  that way we don't need to redo this work next week.